Makefile

```diff
@@ -221,3 +222,44 @@ help:
 	@printf "\n%s\n\n" "usage: make <target>"
 	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[0;36m%-30s\033[0m %s\n", $$1, $$2}'
 
+
+# Utility targets
+#######################
+
+jq-installed:
+ifeq ($(OS),Darwin)
+	# Skipping installation of jq for local dev on Mac.
+	# You can install via 'brew install jq' the following command if needed.
+else
+	which jq ; if [ $$? -eq 1 ] ; then sudo apt-get install -y jq ; fi
```

Comment:
Nit: I think this is a little easier to read.

```
if which jq; then sudo apt-get install -y jq ; fi
```

Reply:
I'm AFK, but wouldn't your rewrite do the opposite? I'm looking to install jq if it isn't found (error code 1).

Reply:
Would you prefer

```
if which jq ; then ; else sudo apt-get install -y jq ; fi
```

Reply:
whoops, you're right! This is what I _meant_ to say :)
```
if ! which jq; then sudo apt-get install -y jq ; fi
```
However, it's not a big deal. This just feels cleaner to _me_

Reply:
Done!